airbyte-integrations/connectors/source-smartsheets-2/README.md

```diff
@@ -0,0 +1,166 @@
+# Smartsheets 2 Source
```

Comment:
[Suggestion] We could have a better name here like `Smarsheets Canonical`(not sure).